sdk/python/foundation-models/system/content-safety/docker_env/conda_dependencies.yaml

```diff
@@ -0,0 +1,16 @@
+channels:
+- conda-forge
+dependencies:
+- python=3.8.16
+- pip<=23.1.2
+- pip:
+  - mlflow<3,>=2.3
+  - azureml-evaluate-mlflow==0.0.14.post1
```

Comment:
This is pretty old. We are now at 0.0.18.post1

Reply:
I grab from model, before I find out a way to dynamically get dependency, this is going to be issue.
Do you mind add this in a check list whenever you update model dependency?